content/post/sonar-goes-live.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "SONAR goes live"
+date: 2021-05-05
+draft: false
+tags: ["project", "shared portal", "dedicated portal", "open access"]
+---
+
+After the successful conclusion of the SONAR development project, the service is ready for its go-live and does so with its very first client : [Haute école pédagogique du Valais](http://www.hepvs.ch/). As RERO+ has been officially established as a private foundation, this is an exciting time to get a glimpse into the future of SONAR.
+
+<!--more-->
+
+## A federating project
+
+Part of the P-5 program (Scientific information: access, processing and safeguarding) of [swissuniversities](https://www.swissuniversities.ch/), SONAR was developed in order to provide a national platform for Swiss open access scientific publications. Four partners from all language areas of Switzerland collaborated during two years to carry out the project : [RERO](https://www.rero.ch/), [HES-SO](https://www.hes-so.ch/) / [HEG Genève](https://www.hesge.ch/heg/), [FH Graubünden](https://www.fhgr.ch/), [Università della Svizzera italiana](https://www.usi.ch/).
+
+## SONAR's live services
+
+- **The [global portal](https://sonar.ch/):** this service aggregates all Swiss open access publications through content tracking. This mechanism harvests metadata from national and international repositories and creates a database. For each document with a full-text, a link to the source server (in principle the repository or the editor website) is available.
+
+- **The dedicated portal:** For institutions that need it, SONAR can also act as a dedicated portal to host any type of digital library, not necessarliy open access. This type of portal benefits from an extended customer support and can be extensively customized by the client with its own visual identity as well as specific metadata fields. The [Haute école pédagogique du Valais](lien vers le portail dédié) uses this service and is SONAR's first client to go live.
```

Comment:
Je renommerais ce service en **Institutional Repository as a Service**, en adaptant le texte en fonction et en précisant les deux modalités (shared, dedicated)

Je citerais les fonctionnalités principales:

> The main functionalities includes: publication deposit and validation, search, display, configuration, Switch edu-ID login. They have been publicly tested by various institutions throughout Switzerland and subsequently improved.
> The service operation and business model are well established and known by RERO, thanks to its RERO DOC experience. The service is completely new, among other things because it consists in a multi-tenant infrastructure in the cloud, in the four national languages and built on the latest version of Invenio.

Reply:
J'ai de la peine avec cette présentation. Je trouve que ça floute beaucoup la frontière entre le shared et le dedicated portal, alors que ce sont deux services vraiment très différents, à mon sens. Peut-être que c'est moi qui suis un peu lent mais je n'ai compris que le mois dernier la différence entre ces deux services (sans parler du portail global). J'essaierais de rendre très clair ce qui est proposé pour chaque service.

Le paragraphe qui décrit les fonctionnalités est très flou pour moi. La première phrase, on parle du dedicated, ou du shared ? Et la dernière phrase : le global et/ou le shared ?

Est-ce que le shared portal ne devrait pas être plus rattaché au global portal dans la logique, plutôt que de le présenter avec le dedicated alors que les deux services sont très différents ?